packages/react-vapor/src/components/tables/TableHeaderCell.tsx

```diff
@@ -30,6 +30,9 @@ export interface ITableHeaderCellProps
         ITableHeaderStateProps,
         ITableHeaderCellDispatchProps {}
 
+/**
+ * @deprecated use TableHeaderWithSort instead

```

Comment:
Could you specify the version of deprecation?
It would also be interesting to also add the version number for the deletion (often the next major version)

Reply:
This is deprecated since I don't know when so i'll put the current version even if it's technically a lie :laughing: 

The deletion thing, I don't know if we have a timeline on that? :thinking:  @GermainBergeron